src/components/AddClient/index.js

```diff
@@ -126,6 +130,22 @@ export function AddClient({ setIsFormShowing, clientsData, setClientsData }) {
                     handleBlur={handleBlur}
                     errors={errors}
                   />
+                  <FormMultipleSelect
```

Comment:
Create a bool value that shows if the user is admin or not and show the dropdown based on it
```suggestion
                  {isAdminUser && <FormMultipleSelect
```